Kickstarter-iOS/Views/Cells/PillCell.swift

```diff
@@ -0,0 +1,63 @@
+import Library
+import Prelude
+import UIKit
+
+final class PillCell: UICollectionViewCell, ValueCell {
+  // MARK: - Properties
+
+  private(set) lazy var label = {
+    UILabel(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  // MARK: - Lifecycle
+
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+
+    _ = self.contentView
+      |> \.layoutMargins .~ UIEdgeInsets(topBottom: Styles.gridHalf(2), leftRight: Styles.gridHalf(3))
+
+    _ = (self.label, self.contentView)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToMarginsInParent()
+  }
+
+  required init?(coder _: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  // MARK: - Styles
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self.contentView
+      |> contentViewStyle
+
+    _ = self.label
+      |> labelStyle
+  }
+
+  // MARK: - Configuration
+
+  func configureWith(value: String) {
+    _ = self.label
+      |> \.text .~ value
+  }
+}
+
+// MARK: - Styles
+
+private let contentViewStyle: ViewStyle = { view in
+  view
+    |> checkoutRoundedCornersStyle
+    |> \.backgroundColor .~ UIColor.ksr_green_500.withAlphaComponent(0.06)
+}
+
+private let labelStyle: LabelStyle = { label in
+  label
+    |> \.font .~ UIFont.ksr_body(size: 13).bolded
```

Comment:
@dannyalright and I have been trying to stick to using Apple's default text styles, without size overrides. Maybe `ksr_footnote` would work here?

Reply:
Happy to change...just wanted to understand what was the rationale? The size override still scales properly...and supports Dynamic Type.

Reply:
We're hoping to be able to get to a place sometime where we can remove our custom font initialization:

```
  let font = UIFont.preferredFont(forTextStyle: style)
  let descriptor = font.fontDescriptor
    return UIFont(
      descriptor: descriptor,
      size: ceil(font.pointSize / defaultSize * (size ?? defaultSize))
    )
```

... and just use `UIFont.preferredFont(forTextStyle: style)` out of the box for simplicity and 1:1 consistency with Apple's suggested sizes for a given text style.

Reply:
+1 to @ifbarrera's explanation. Additionally: designers can access Apple's text styles in Sketch. Engineers can see this text style via Abstract/Sketch. Just makes things more straightforward all-round.

Reply:
^^^
CC @cdolm92 @justinswart @garycanino

Reply:
I learned about Apple's Sketch library recently, so parts of Checkout (including these pills) does not take advantage of it. For those cases (until they're updated by designover time), you can reference [Apple's text styles list](https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/typography/) to see what matches most closely to the UI mockup.

Reply:
The way other font sizes resize is unfortunately not identical to what `body` currently does so I'm not really sure which font would match the closest without explicitly specifying the size. Will do some more testing..

Reply:
Alright, resolved this with Danny face to face